### ISSUE TITLE
Splitting ACCESS-OM2 into multiple model names?

### ISSUE THREAD

**Issue by adele-morrison** [ACCESS-NRI/access-nri-intake-catalog#194]
From the COSIMA meeting today:

Currently all resolutions of ACCESS-OM2 show up in the Catalog as the `ACCESS-OM2` model. Usually we only use this name for the 1deg config. Do people think it's worth going through all the metadata.yaml files for ACCESS-OM2-01 and ACCESS-OM2-025 experiments and renaming the model from just ACCESS-OM2? It would make it easier for people to search and find output that is relevant for them if they can filter by model name better.

**Comment by adele-morrison** [comment]
I don't know why there's a bug label on this sorry, I can't edit the labels.

**Comment by marc-white** [comment]
Sticking my initial thoughts in-meeting here:

- Would adding extra 'things' to the Catalog name potentially confuse people, or is this pretty standard nomenclature?
- It seems like resolution would be a convenient thing to be able to search on at the `access-nri-intake-catalog` level (i.e., when you're searching for experiments). Are there any other similar things you'd want to search on at the experiment level that you can't search on currently?

**Comment by adele-morrison** [comment]
Currently there is an `ACCESS-OM2-025` model name that only contains the CMIP 1/4deg runs. But the non-CMIP 1/4 deg runs are labelled `ACCESS-OM2`. Confusingly the 1deg CMIP runs are labelled `ACCESS-OM2` along with all the 1/10th runs etc.

**Comment by dougiesquire** [comment]
Just noting that the model name is whatever is specified in the `model` field in the `metadata.yaml` file. So changing these is as simple as updating the `metadata.yaml` files.

**Comment by adele-morrison** [comment]
Yep, got that @dougiesquire. I was checking that we aren't missing some reason why this wasn't done in the first place? If not, I suggest we go ahead and rename all higher res `ACCESS-OM2` model names in the `metadata.yaml`s.

**Comment by adele-morrison** [comment]
Ok, I renamed the model name to `ACCESS-OM2-01` in the `metdata.yaml` of all the ACCESS-OM2-01 experiments I have write access to.

@aekiss, @hrsdawson @mauricehuguenin @pedrocol @willaguiar can you do the same please? Anyone else we need to tag?

**Comment by adele-morrison** [comment]
I guess once that's done we'll need to rerun all the Intake stores for those experiments.

**Comment by dougiesquire** [comment]
> I guess once that's done we'll need to rerun all the Intake stores for those experiments.

Yes, a new version of the catalog will need to be built

**Comment by anton-seaice** [comment]
Is this the field:

`model: access-om2-01` 

https://github.com/ACCESS-NRI/access-om2-configs/blob/83885a45017b3a34ea4fad15d155449c7217ad70/metadata.yaml#L28

It looks like its already been changed in the release config ?

**Comment by adele-morrison** [comment]
> Is this the field:
> `model: access-om2-01`

Yes that's it. Is it case sensitive? All of the existing ones are in capitals
`model: ACCESS-OM2-01`

**Comment by anton-seaice** [comment]
Oh - we appear to have not done that consistently:

`model: ACCESS-OM2` is in some of the 025deg configs

https://github.com/ACCESS-NRI/access-om2-configs/blob/da5f7965b41b516c2ca0eaa4b235d81d03e19f89/metadata.yaml#L33

Hopeing @marc-white can confirm about the case-sensitivity ?

**Comment by aekiss** [comment]
> * It seems like resolution would be a convenient thing to be able to search on at the `access-nri-intake-catalog` level (i.e., when you're searching for experiments). Are there any other similar things you'd want to search on at the experiment level that you can't search on currently?

It could be handy to be able search on the items in the [`keywords` field in `metadata.yaml`](https://github.com/ACCESS-NRI/access-om2-configs/blob/83885a45017b3a34ea4fad15d155449c7217ad70/metadata.yaml#L15-L19) such as forcing dataset (JRA55do, ERA5) and repeat-year/interannual.

**Comment by marc-white** [comment]
> Oh - we appear to have not done that consistently:
> 
> `model: ACCESS-OM2` is in some of the 025deg configs
> 
> https://github.com/ACCESS-NRI/access-om2-configs/blob/da5f7965b41b516c2ca0eaa4b235d81d03e19f89/metadata.yaml#L33
> 
> Hopeing @marc-white can confirm about the case-sensitivity ?



> Oh - we appear to have not done that consistently:
> 
> `model: ACCESS-OM2` is in some of the 025deg configs
> 
> https://github.com/ACCESS-NRI/access-om2-configs/blob/da5f7965b41b516c2ca0eaa4b235d81d03e19f89/metadata.yaml#L33
> 
> Hopeing @marc-white can confirm about the case-sensitivity ?

I've not yet been able to find anywhere where `access-nri-intake-catalog` or `intake-esm` where it does case modification on the way in, so I think it's likely to be case-sensitive (although it doesn't help that every example given has fully capitalized model names anyway).

**Comment by aekiss** [comment]
I've fixed these ones
```
/g/data/cj50/access-om2/raw-output/access-om2-01/01deg_jra55v140_iaf*/metadata.yaml
/g/data/ik11/outputs/access-om2-01/01deg_jra55v140_iaf_cycle4_jra55v150_extension/metadata.yaml
```

**Comment by aekiss** [comment]
We'll also need to fix `metadata.yaml` in any actively running control dirs, and fix the capitalisation in https://github.com/ACCESS-NRI/access-om2-configs

**Comment by anton-seaice** [comment]
> Do people think it's worth going through all the metadata.yaml files for ACCESS-OM2-01 and ACCESS-OM2-025 experiments and renaming the model from just ACCESS-OM2? It would make it easier for people to search and find output that is relevant for them if they can filter by model name better.

On this questions @jo-basevi asks if we should distinguish BGC vs not BGC ? (e.g. should there also be ACCESS-OM2-01-BGC , ACCESS-OM2-025-BGC etc)

**Comment by anton-seaice** [comment]
@adele-morrison @aekiss @AndyHoggANU 

Any preference on whether `ACCESS-OM2-01-BGC` should be a different model in the intake catalog than `ACCESS-OM2-01` ?

My feeling is not, because I think users look at BGC runs for non BGC things a bit ?

**Comment by AndyHoggANU** [comment]
No, I don't think we should call the BGC model a different model in the intake catalog.

**Comment by dougiesquire** [comment]
I agree no. In fact, I would argue that the model should not include the resolution - that is configuration-specific, not model-specific. But I think that ship has sailed.

**Comment by aidanheerdegen** [comment]
> In fact, I would argue that the model should not include the resolution - that is configuration-specific, not model-specific. But I think that ship has sailed.

I agree. We have a `nominal_resolution` field for this purpose. Not sure how it is utilised in the catalogue.

https://github.com/ACCESS-NRI/access-om2-configs/blob/release-01deg_jra55_iaf_bgc/metadata.yaml#L34
